docs/UserGuide.md

```diff
@@ -31,104 +31,238 @@
 
 > **Command Format**
 > * Words in `UPPER_CASE` are the parameters.
-> * Items in `SQUARE_BRACKETS` are optional.
+> * Items in `[SQUARE_BRACKETS]` are optional.
 > * Items with `...` after them can have multiple instances.
-> * The order of parameters is fixed.
+> * The order of parameters is not fixed.
 
 #### Viewing help : `help`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a task or event: `add`
 
-> Persons can have any number of tags (including 0)
+Adds a task to Lifekeeper<br>
+Format: `add TASK_NAME [d/DEADLINE] [p/PRIORITY_LEVEL] [r/REMINDER] [t/TAG]...` 
+
+> Tasks can have any number of tags (including 0)
+
+`DEADLINE` accepts `Date Time` format input and variable inputs namely:
+* `tomorrow [TIME]`
+* `today [TIME]`
+
+`PRIORITY_LEVEL` has to be an integer equal or larger than 1, with `1` being the top priority.
+
+`REMINDER` accepts `Date Time` format input and variable inputs namely:
```

Comment:
reminder is a useful feature to have, however, you can make it a lower priority item on your schedule. aim to first finish the must-have features :-)